src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,63 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+
 public class WorkWithFile {
+    private static final int OPERATION_TYPE_INDEX = 0;
+    private static final int AMOUNT_INDEX = 1;
 
     public void getStatistic(String fromFileName, String toFileName) {
+        String[] splitInformationFromFile;
+        StringBuilder builder = new StringBuilder();
+        try (BufferedReader reader = new BufferedReader(new FileReader(fromFileName))) {
+            String value = reader.readLine();
+            while (value != null) {
+                builder.append(value).append(System.lineSeparator());
+                value = reader.readLine();
+            }
+            splitInformationFromFile = builder.toString().split(System.lineSeparator());
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read a file " + fromFileName, e);
+        }
+        String report = createReport(splitInformationFromFile);
+        File newFile = new File(toFileName);
+        WorkWithFile file = new WorkWithFile();
+        file.writeToFile(newFile, report);
```

Comment:
```suggestion
        String dataFromFile = readFromFile(fromFileName);
        String report = createReport(splitInformationFromFile);
        writeToFile(toFileName, report);
```
lets move logic to read data to a separate method
and create file in writeToFile method